demo/SFSW2_project_descriptions.R

```diff
@@ -71,10 +71,10 @@ project_paths <- list(
 
   # Path from where external data are extraced
   dir_external = dir_ex <- if (identical(opt_platform[["host"]], "local")) {
-      file.path("/Volumes", "YOURDRIVE", "BigData", "GIS", "Data")
-    } else if (identical(opt_platform[["host"]], "hpc")) {
-      file.path("/home", "YOURDRIVE", "BigData", "GIS", "Data")
-    },
+    file.path("/media/natemccauslin/SOILWAT_DATA/GIS/Data")
```

Comment:
These needs to remain as: file.path("/Volumes", "YOURDRIVE", "BigData", "GIS", "Data"). This is just a dummy path, obviously, and your local path should not be submitted.